test/browser/ui.spec.js

```diff
@@ -19,7 +19,7 @@ describe('$.jQuery', function () {
 });
 
 // another generic describe block to verify it is absent
-// when greeping on $.jQuery
+// when greping on $.jQuery
```

Comment:
I'm also fairly sure that "grepping" is even better than "greping" per typical English language consonant-in-middle-of-word rules, unless using some kind of quote or codeblock-backtick to separate the `grep` from the "ing", but, eh, anything's better than "greeping" (:laughing:).

Reply:
Err, yeah. I have no idea why I picked that spelling. Will fix.